cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/service/executor/DatasetAdminOpHTTPHandler.java

```diff
@@ -61,25 +61,32 @@ public DatasetAdminOpHTTPHandler(Authenticator authenticator, RemoteDatasetFrame
     this.dsFramework = dsFramework;
   }
 
+  @GET
+  @Path("/data/datasets/flipper")
+  public void foo(HttpRequest request, HttpResponder responder, @PathParam("namespace-id") String namespaceId) {
+    responder.sendString(HttpResponseStatus.OK, "YAY!");
+  }
+
   @POST
   @Path("/data/datasets/{name}/admin/exists")
```

Comment:
Just FYI: These endpoints don't need to start with a "/" since `netty-http` implicitly adds it.